docs/adr/0003-worldwide-organisation-multi-part-content.md

```diff
@@ -0,0 +1,44 @@
+# 3. Using multi-part content for Worldwide Organisations
+
+Date: 2024-06-04
+
+## Context
+
+Whilst converting Worldwide Organisations from being non-editionable to editionable entities in Whitehall Publisher, an investigation was carried out into the best way of modelling the child pages that belong to the organisation when sent to Publishing API (and therefore Content Store).
+
+There are two ways that a piece of content that is spread over several pages may currently be represented on GOV.UK:
+
+1. Multiple pages (known as “parts”) on GOV.UK that are published as a single content item (i.e. all the sub-pages are in one content item, with a single content item). There are two existing uses of these:
+   * Mainstream guides
+   * Travel advice
+
+1. Multiple pages on GOV.UK that form part of the same piece of guidance, but are published as multiple content items (i.e. each part is in a separate content items). There are many example of this:
```

Comment:
Numbering tweak - you've got two '1s'!

Reply:
Markdown coverts these into a numbered list when rendered 🤓.  Putting everything as a '1' means you don't need to renumber if new lines are added later.